scaletest/dashboard/rolltable.go

```diff
@@ -0,0 +1,304 @@
+package dashboard
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/coder/coder/codersdk"
+)
+
+// allActions is a table of actions to perform.
+// D&D nerds will feel right at home here :-)
+// Note that the order of the table is important!
+// Entries must be in ascending order.
+var allActions rollTable = []rollTableEntry{
+	{0, fetchWorkspaces, "fetch workspaces"},
+	{1, fetchUsers, "fetch users"},
+	{2, fetchTemplates, "fetch templates"},
+	{3, authCheckAsOwner, "authcheck owner"},
```

Comment:
Might be worth getting the list of auth checks the UI does all the time. Not needed in this PR, but the FE does like 10ish checks on each load.